.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   # Install all dependencies and run check tasks.
   build_and_test:
     docker:
-      - image: circleci/php:8.0-cli-node
+      - image: cimg/php:8.1
```

Comment:
Seems we just needed to update to newer PHP as I think it was here in Finder library where it failed

    $this->dirs = array_merge($this->dirs, ...$resolvedDirs);
    
circleci/php is deprecated so moving to cimg/php ( https://circleci.com/developer/images/image/cimg/php ) 

There was no cli tag and node is not needed. At least I don't know why it would be needed.